pubsub/google/cloud/pubsub_v1/gapic/publisher_client.py

```diff
@@ -51,10 +51,10 @@
 
 
 # TODO: remove conditional import after Python 2 support is dropped
-if six.PY3:
```

Comment:
I didn't notice this when this change was being reviewed, but we'd have to make this change to the templates in gapic-generator to make this permanent. I don't anticipate us staying on the monolith(or supporting Python 2) past the first half of this year.

@plamut Would you be alright with letting this change be reverted? A synth replace is also an option if you feel it is important for Pub/Sub in particular.

Mostly trying to clear out synth PRs before repo split on Friday :)

Reply:
@busunkim96 You mean reverting the change back to checking "if PY3"?

While this fix by an external contributor was welcome, it won't be applicable in the foreseeable future, as Python 4 has not been announced yet, and it's nearly certain that Python 2 support will be dropped way before that.

Feel free to do anything that makes the job on your part easier. :)